### PR TITLE
Update raspberry_pi.md

### PR DESCRIPTION
**Pull request by SalimTerryLi**
All things have been translated into Chinese.

### CODE REVIEW

**Comment by SalimTerryLi** [comment]
Sorry. I've been so careless that there are many problems...

**Comment by mhkabir** [comment]
No issue. Please fix them, and we'll merge it :)

**Comment by mhkabir** [comment]
@FantasyJXF please review

File: zh/flight_controller/raspberry_pi.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ translated_sha: 95b39d747851dd01c1fe5d36b24e59ec865e323e
 
 ### OS Image系统镜像
 
-  使用[Emlid RT Raspbian image](http://docs.emlid.com/navio/Downloads/Real-time-Linux-RPi2/)这个前期配置好的有效的PX4树莓派镜像。这个镜像默认最大化的事先配置了程序。
+  使用[Emlid RT Raspbian image](http://docs.emlid.com/navio/Downloads/Real-time-Linux-RPi2/)这个前期配置好的有效的PX4树莓派镜像。这个镜像默认最大化的事先配置了程序。specifically
 
-> **Important**: make sure not to upgrade the system (more specifically the kernel). By upgrading, a new kernel can get installed which lacks the necessary HW support (you can check with `ls /sys/class/pwm`, the directory should not be empty).
+> **重要**: 确保不要升级系统 （更确切的说是内核）。 升级后，一个缺少必要硬件支持的新内核将被安装。(你可以通过 `ls /sys/class/pwm`命令查看，这个目录不应当是空的)。
```
Comment (hamishwillee) [inline zh/flight_controller/raspberry_pi.md:17]: Don't translate the first bits  like:  `> **Important** ` - this is a hint to the plugin to render the text in  a certain colour box with a certain icon. The text does not appear in built version (though it does in github).

File: zh/flight_controller/raspberry_pi.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ translated_sha: 95b39d747851dd01c1fe5d36b24e59ec865e323e
 
 ### OS Image系统镜像
 
-  使用[Emlid RT Raspbian image](http://docs.emlid.com/navio/Downloads/Real-time-Linux-RPi2/)这个前期配置好的有效的PX4树莓派镜像。这个镜像默认最大化的事先配置了程序。
+  使用[Emlid RT Raspbian image](http://docs.emlid.com/navio/Downloads/Real-time-Linux-RPi2/)这个前期配置好的有效的PX4树莓派镜像。这个镜像默认最大化的事先配置了程序。specifically
```
Comment (hamishwillee) [inline zh/flight_controller/raspberry_pi.md:15]: Text at end "specifically" is in English.

File: zh/flight_controller/raspberry_pi.md [hunk 3]
```diff
@@ -99,7 +99,7 @@ sudo /etc/init.d/avahi-daemon restart
 You might have to add .local to the hostname to discover it.
```
Comment (hamishwillee) [inline zh/flight_controller/raspberry_pi.md:99]: English sentence here.

**Review by SalimTerryLi — COMMENTED** [review]
OK, I‘ ll fix it.

**Comment by FantasyJXF** [comment]
Sprey for that . I' ll check it, I'd told them to first pull the request to me,Maybe I should emphasize it

**Comment by SalimTerryLi** [comment]
@FantasyJXF Ohh... What should I do?

**Comment by FantasyJXF** [comment]
please  contact me QQ  931026752

**Pull request closed without merging**